src/components/PhotoGrid/PhotoGrid.js

```diff
@@ -1,17 +1,16 @@
 import styles from "./PhotoGrid.module.css";
-import trainPhoto from "../../Images/photogrid/photogrid_1.png";
-import cloudsPhoto from "../../Images/photogrid/photogrid_2.png";
-import wolfPhoto from "../../Images/photogrid/photogrid_3.png";
-import forestPhoto from "../../Images/photogrid/photogrid_4.png";
-import carPhoto from "../../Images/photogrid/photogrid_5.png";
-import beachPhoto from "../../Images/photogrid/photogrid_6.png";
-import mountainsPhoto from "../../Images/photogrid/photogrid_7.png";
-import roadSignPhoto from "../../Images/photogrid/photogrid_8.png";
-import mistyPhoto from "../../Images/photogrid/photogrid_9.png";
-import treesPhoto from "../../Images/photogrid/photogrid_10.png";
-import cliffPhoto from "../../Images/photogrid/photogrid_11.png";
-import topsPhoto from "../../Images/photogrid/photogrid_12.png";
-
+import trainPhoto from "../../images/photogrid/photogrid_1.png";
+import cloudsPhoto from "../../images/photogrid/photogrid_2.png";
+import wolfPhoto from "../../images/photogrid/photogrid_3.png";
+import forestPhoto from "../../images/photogrid/photogrid_4.png";
+import carPhoto from "../../images/photogrid/photogrid_5.png";
+import beachPhoto from "../../images/photogrid/photogrid_6.png";
+import mountainsPhoto from "../../images/photogrid/photogrid_7.png";
+import roadSignPhoto from "../../images/photogrid/photogrid_8.png";
+import mistyPhoto from "../../images/photogrid/photogrid_9.png";
+import treesPhoto from "../../images/photogrid/photogrid_10.png";
+import cliffPhoto from "../../images/photogrid/photogrid_11.png";
+import topsPhoto from "../../images/photogrid/photogrid_12.png";
```

Comment:
These changes are not relevant to `Cover` section. You can ignore this comment for now, but please keep in mind that PR name and its changes should be relevant to each other.

Reply:
understood